_data/committee/national_chapters.yml

```diff
@@ -31,3 +34,21 @@ nlRSE:
   long_name: The Netherlands Research Software Engineer community
   internal: /contact/chapters/#nlRSE
   twitter: nl_rse
+  github: nl-rse
+nordicRSE:
+  name: Nordic-RSE
+  website: https://nordic-rse.org/
+  avatar: https://nordic-rse.org/img/logo.png
+  email: contact@nordic-rse.org
+  long_name: Nordic research software engineers
+  internal: /contact/chapters/#nordicRSE
+  twitter: nordic_rse
+  github: nordic-rse
+aunzRSE:
+  name: RSE-AUNZ
+  website: https://rse-aunz.org
```

Comment:
```suggestion
  website: https://rse-aunz.github.io
```
For https://rse-aunz.org I am getting an alert of a potential security risk. Is there a reason why you chose this URL @florianthiery?

![image](https://user-images.githubusercontent.com/9960249/84368310-aef69200-abd5-11ea-8ef1-4ac007ff05b2.png)